Stagger the execution of the collection import tasks to minimize dead…

dbernstein opened the pull request:
…locks.

## Description
This update introduces a 5 second delay between the execution of each axis collection import task in order to reduce the probability of a simultaneous update of a particular book on CMs where there are multiple axis collections with overlapping contents.
<!--- Describe your changes -->

## Motivation and Context
https://ebce-lyrasis.atlassian.net/browse/PP-2233
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

## How Has This Been Tested?

<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Checklist

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.

jonathangreen commented on src/palace/manager/celery/tasks/axis.py:
```diff
@@ -49,6 +49,13 @@ def import_all_collections(
             )
             list_identifiers_for_import.apply_async(
                 kwargs={"collection_id": collection.id},
+                eta=utc_now()
+                + timedelta(
+                    seconds=count * 5
+                ),  # stagger the execution of the collection import
+                # in order to minimize chance of deadlocks caused by
+                # simultaneous updates to the metadata when CM has multiple
+                # axis collections configured with overlapping content
```
Minor: Instead of having to work with a `timedelta` here, you could just use the [`countdown` parameter](https://docs.celeryq.dev/en/stable/userguide/calling.html#eta-and-countdown).
```suggestion
                delay=count * 5,  # stagger the execution of the collection import
                # in order to minimize chance of deadlocks caused by
                # simultaneous updates to the metadata when CM has multiple
                # axis collections configured with overlapping content
```

dbernstein replied:
I'll give it a shot - but it looks like we should be using `countdown` not `delay` unless I'm missing something.

jonathangreen reviewed (approved):
🚀

dbernstein commented:
@jonathangreen : I updated this one to use countdown instead of ETA, employed a similar changed for the reap_all_collections task and updated the tests to reflect these changes in parameters.
